Style student group component

SteveLorentzen opened the pull request.

SteveLorentzen committed:
Fixed small css typos

SteveLorentzen committed:
Worked on styling the StudentGroup component for mobile

SteveLorentzen committed:
Changed border-radius of NameDisplay component

trevorfehrman commented on src/components/EditableStudentGroupName.tsx:
```diff
@@ -13,6 +13,10 @@ const EditableStudentGroupName: React.FC<EditableStudentGroupNameProps> = ({
 }) => {
   const editStudentGroupNameHandler = (value: string) => {
     console.log(value)
+    if (value.length > 22) {
```
is this just a custom validation?  better to validate through react-hook-form, no?

SteveLorentzen replied:
Yeah, if I end up doing this.  Do you think it's an okay idea to limit the characters in a group name for display purposes?

SteveLorentzen replied:
Okay, after a long struggle I got the onSubmit working for the Editable chakra input but now the validation is making the input behave weird (keypresses are not registering consistently as error messages are displayed).  I'm starting to wonder if using these two libraries together is too big of a pain in this particular case.  Since the Editable component is just a single input, would it be okay to just do my own validation in this case?  I would still use chakra for the error messages so the styling would be consistent.  Then I would use react-hook-form for any real forms, like with normal inputs and a submit button.  What do you think?

SteveLorentzen committed:
Used react-hook-form to validate
I can get everything to work the way I want it except for the onSubmit
If I use onBlur, the the submit handler works
I just want the form to be submitted if the user presses enter or if the input is blurred

SteveLorentzen committed:
Got the onSubmit event working but now there's some weird error handling behavior :(

SteveLorentzen committed:
Styled several other components and switched to custom validation on Editable chakra component
